vendor/index.yaml

```diff
@@ -1969,3 +1969,9 @@ vendors:
     logo: koidra-logo.svg
     social:
       linkedin: https://www.linkedin.com/company/koidra/
+      
+  - id: open-boat-projects
+    name: Open Boat Projects
+    description: Open Boat Projects is a platform on which water sports enthusiasts present their self-constructed DIY projects around boating. The focus is on the idea of open source and open hardware as well as the exchange of ideas.
+    website: https://open-boat-projects.org
+    logo: obp_logo.jpg
```

Comment:
change logo extension to .png, in index .jpg is not valid

Reply:
@norbert-walter don't forget to change the extension to your logo and here to .png